packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -55,24 +55,34 @@ export class DataSyncPlugin extends GraphbackPlugin {
 
             return schema;
         }
-
         models.forEach((model: ModelDefinition) => {
-            const SyncResultList = new GraphQLObjectType({
-                name: `${model.graphqlType.name}DeltaList`,
+            
+            const modelName = model.graphqlType.name;
+            const modifiedType = schemaComposer.getOTC(modelName);
+            const entries = Object.entries(modifiedType.getFields()).filter((e: [string, ObjectTypeComposerFieldConfig<any, unknown, any>]) => {
+                return parseRelationshipAnnotation(e[1].description) === undefined
+            })
+
+            const fields = Object.assign({}, ...Array.from(entries, ([k, v]: [string, any]) => ({ [k]: v })));
+
+            schemaComposer.createObjectTC(getDeltaType(modelName)).addFields({
+                ...fields,
+                deleted: 'Boolean',
```

Comment:
_deleted could be better to not clash with existing names

Reply:
Done